uwblueprint/focus-on-nature
# feat: Set up Chakra UI and style login page

## Notion ticket link

[Staff Tool: Login Page](https://www.notion.so/uwblueprintexecs/Staff-Tool-Login-Page-dcb689f07cbd445b9cced5eeba7334cf)

## Implementation description
* Installed Chakra UI, react-icon libraries 
* Set up design system themes for font styles and colours based on [Figma design system](https://www.figma.com/file/9eisz5j6d4KJEd3KkhijvI/FON-%2F-Design-System?node-id=12%3A2)
* Edited login route to be `/admin/login` instead of `/login`
* Styled login page


## Steps to test
1. Go to `localhost:3000/admin/login`
2. Make sure style matches designs
3. Log in using a valid email (should be able to log in)
4. Log in using a non FON email (invalid error message should show and can't log in)
5. Log in using an inactive email (inactive error message should show and can't log in)

Figma designs for login: 
https://www.figma.com/file/uRMwPpIaSgsMGdbJzZziMA/FON-%2F-Management-Experience?node-id=2%3A4



## What should reviewers focus on?
* Make sure text styles and colours are accurate by double checking with the design system on Figma


## Checklist
